pkg/backup/backup.go

```diff
@@ -334,9 +336,13 @@ func (kb *kubernetesBackupper) BackupWithResolvers(log logrus.FieldLogger,
 			case <-ticker.C:
 				if lastUpdate != nil {
 					backupRequest.Status.Progress = &velerov1api.BackupProgress{TotalItems: lastUpdate.totalItems, ItemsBackedUp: lastUpdate.itemsBackedUp}
-					original := backupRequest.Backup.DeepCopy()
-					backupRequest.Backup.Status.Progress = &velerov1api.BackupProgress{TotalItems: lastUpdate.totalItems, ItemsBackedUp: lastUpdate.itemsBackedUp}
-					if err := kube.PatchResource(original, backupRequest.Backup, kb.kbClient); err != nil {
+					updated := backupRequest.Backup.DeepCopy()
```

Comment:
This line needs some modification. Please swap it with the previous line.

```suggestion
if lastUpdate != nil {
					updated := backupRequest.Backup.DeepCopy()
					backupRequest.Status.Progress = &velerov1api.BackupProgress{TotalItems: lastUpdate.totalItems, ItemsBackedUp: lastUpdate.itemsBackedUp}
```

Reply:
Done